### PR TITLE
Enhancement: Upgrade composer-normalize-action to 0.8.0

### PR DESCRIPTION
**Pull request by OskarStark**
This PR

* [x] upgrades version to `0.8.0`
* [x] uses new namespace `ergebnis/....`

cc @localheinz

### CODE REVIEW

**Review by localheinz — APPROVED** [review]
👍

**Comment by OskarStark** [comment]
@localheinz any idea why this can't be pulled? 🤔

**Comment by localheinz** [comment]
@OskarStark 

Ha, looks like this is affected by recent changes to Docker's image retainment policy. Hang on!

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -27,7 +27,7 @@ jobs:
               uses: actions/checkout@v2
 
             - name: "Run composer-normalize"
-              uses: docker://localheinz/composer-normalize-action:0.5.2
+              uses: docker://localheinz/composer-normalize-action:0.8.0
```
Comment (localheinz) [inline .github/workflows/ci.yaml:30]:
```suggestion
              uses: docker://ergebnis/composer-normalize-action:0.8.0
```

File: templates/project/.github/workflows/lint.yaml [hunk 2]
```diff
@@ -31,7 +31,7 @@ jobs:
               uses: actions/checkout@v2
 
             - name: "Run composer-normalize"
-              uses: docker://localheinz/composer-normalize-action:0.5.2
+              uses: docker://localheinz/composer-normalize-action:0.8.0
```
Comment (localheinz) [inline templates/project/.github/workflows/lint.yaml:34]:
```suggestion
              uses: docker://ergebnis/composer-normalize-action:0.8.0
```

**Comment by greg0ire** [comment]
> Ha, looks like this is affected by recent changes to Docker's image retainment policy. Hang on!

Yup, a lot of images are missing here: https://hub.docker.com/r/localheinz/composer-normalize-action/tags

**Review by localheinz — COMMENTED** [review]
Perhaps this helps?

**Comment by OskarStark** [comment]
> Perhaps this helps?

Lets see... ⏳

**Pull request merged**